src/main/java/com/choelinska/horrorlibz/services/UserService.java

```diff
@@ -0,0 +1,12 @@
+package com.choelinska.horrorlibz.services;
+
+import com.choelinska.horrorlibz.model.dto.UserTo;
+import com.choelinska.horrorlibz.model.entities.UserEntity;
+
+
+import java.util.List;
+
+public interface UserService {
+    public void saveUser(UserTo user);
+    List<UserEntity> sortUser();
```

Comment:
Please remove the sort function irrelevant to user registration from this branch.